deployments/Makefile

```diff
@@ -0,0 +1,35 @@
+CLUSTER_NAME = "fondant-cluster"
+
+KUBEFLOW_PIPELINE_VERSION = 1.8.5
+
+
+gcp: setup-gcp-services provision-gcp-cluster authenticate-gcp-cluster install-kfp
+
+setup-gcp-services:
+	gcloud services enable \
+	compute.googleapis.com \
+	container.googleapis.com \
+	iam.googleapis.com \
+	servicemanagement.googleapis.com \
+	cloudresourcemanager.googleapis.com \
+	ml.googleapis.com \
+	iap.googleapis.com \
+	sqladmin.googleapis.com \
+	meshconfig.googleapis.com \
+	krmapihosting.googleapis.com
+
+provision-gcp-cluster:
+	gcloud container clusters create ${CLUSTER_NAME} \
+     --machine-type "e2-standard-2" \
+     --scopes "cloud-platform"
+
+authenticate-gcp-cluster:
+	gcloud container clusters get-credentials ${CLUSTER_NAME} \
+
+install-kfp:
+	kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=${KUBEFLOW_PIPELINE_VERSION}"
+	kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+	kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=${KUBEFLOW_PIPELINE_VERSION}"
```

Comment:
The installation here is using the standard `dev`environment. We might want to switch to the GCP one to be able to store the kfp artifact (manifests in this case) in a persistent storage. There are other advantages as well such as persistent storage of pipeline runs on SQL but we can skip that. 
Kubeflow pipelines also had custom enviroments for different clouds: 
https://github.com/kubeflow/pipelines/tree/sdk/release-1.8/manifests/kustomize/env

Links: 
https://github.com/ml6team/Express-infra/tree/main/kfp-setup
https://www.kubeflow.org/docs/components/pipelines/v1/installation/standalone-deployment/#:~:text=to%20do%20anything.-,Customizing%20Kubeflow%20Pipelines,-Kubeflow%20Pipelines%20can

Reply:
So I decided against it since you need to setup a SQL database which is not cheap. I did add a link to this so the user can still go for custom deployments if they want to.